Mono.Addins/Mono.Addins/RuntimeAddin.cs

```diff
@@ -341,7 +341,43 @@ public Type GetType (string typeName, bool throwIfNotFound)
 				throw new InvalidOperationException ("Type '" + typeName + "' not found in add-in '" + id + "'");
 			return null;
 		}
-		
+
+		internal Type GetType (string typeName, string assemblyName, bool throwIfNotFound)
+		{
+			EnsureAssembliesLoaded ();
+
+			// Look in the addin assemblies
+
+			if (assemblyName == null || assemblyName == GetType ().Assembly.FullName)
+			{
```

Comment:
Minor: Formatting here does not match the rest of the project. The curly braces are on the same line as the if statements elsewhere.

Reply:
:+1: It was importing my editorconfig from the repo which submoduled it.